contracts/Certificate.sol

```diff
@@ -58,6 +58,51 @@ contract Certificate is
    */
   Removal private _removal;
 
+  /**
+   * @notice Emitted on updating `_removalBalancesOfCertificate`.
+   * @param certificatedId The certificate to connected to the removal.
+   * @param removalId The removal to update the balance for.
+   * @param isAdded Whether the amount was added or removed.
+   * @param amount The amount to added or removed for the certificate.
+   */
+  event RemovalBalancesOfCertificateUpdated(
+    uint256 indexed certificatedId,
+    uint256 indexed removalId,
+    bool indexed isAdded,
+    uint256 amount
```

Comment:
For something like this, would it be prefereable to use two different events (IE - RemovalBalancesOfCertificateAdded and RemovalBalancesOfCertificateSubtracted), and then we remove the `isAdded` param and index `amount`? Or is it better to have a single event that can return all updates to this state (but without the ability to index on a particular amount)?

Reply:
A single event is preferable where possible as events are expensive

Would be best to stay consistent with event tense (e.g., OZ uses present tense `Transfer`). I think this event should be called `Release`

The flipside of this case is already covered (in `_receiveRemovalBatch` is already covered by ERC721.Mint)

Reply:
not sure I see the value in making these general purpose with the `isAdded` arg